packages/core/src/sims/melee/drg/drg_top_sim.ts

```diff
@@ -0,0 +1,299 @@
+import { Ability, OgcdAbility, Buff, SimSettings, SimSpec, Simulation } from "@xivgear/core/sims/sim_types";
+import { CycleProcessor, CycleSimResult, ExternalCycleSettings, MultiCycleSettings, AbilityUseResult, Rotation, PreDmgAbilityUseRecordUnf, CycleSimResultFull } from "@xivgear/core/sims/cycle_sim";
+import { potionMaxStr } from "@xivgear/core/sims/common/potion";
+import DRGGauge from "./drg_gauge";
+import { DrgAbility, DrgGcdAbility, DRGExtraData } from "./drg_types";
+import * as Actions from './drg_actions';
+import * as Buffs from './drg_buffs';
+import { BaseMultiCycleSim } from "@xivgear/core/sims/processors/sim_processors";
+import { CharacterGearSet } from "../../../gear";
+
+export interface DRGTopSimSettings extends SimSettings {
+
+}
+
+export interface DRGTopSimResult extends CycleSimResult {
+
+}
+
+export interface DRGTopSettingsExternal extends ExternalCycleSettings<DRGTopSimSettings> {
+
+}
+
+export const drgTopSimSpec: SimSpec<DrgTopSim, DRGTopSettingsExternal> = {
+    stub: "drg-top-sim",
+    displayName: "DRG Ultimate LvL 90 Sim",
+    makeNewSimInstance: function (): DrgTopSim {
+        return new DrgTopSim();
+    },
+    loadSavedSimInstance: function (exported: DRGTopSettingsExternal): DrgTopSim {
+        return new DrgTopSim(exported);
+    },
+    supportedJobs: ["DRG"],
+    supportedLevels: [100],
```

Comment:
If this is a level 90 sim, then the supported level should be 90.

Reply:
```suggestion
    supportedLevels: [90],
```

^ should use that feature